api/net/Areas/Subscriber/Controllers/FolderController.cs

```diff
@@ -132,8 +131,13 @@ public IActionResult Add(FolderModel model)
     [SwaggerOperation(Tags = new[] { "Folder" })]
     public IActionResult Update(FolderModel model)
     {
+        var username = User.GetUsername() ?? throw new NotAuthorizedException("Username is missing");
```

Comment:
Fixed permission issue.  @jdtoombs We can't allow users to edit folders they don't own.